server/test/repository/ProgramRepositoryTest.java

```diff
@@ -58,13 +59,15 @@ public void setup() {
     versionRepo = instanceOf(VersionRepository.class);
     mockSettingsManifest = Mockito.mock(SettingsManifest.class);
     programCache = instanceOf(SyncCacheApi.class);
+    programDefCache = instanceOf(SyncCacheApi.class);
     versionsByProgramCache = instanceOf(SyncCacheApi.class);
     repo =
         new ProgramRepository(
             instanceOf(DatabaseExecutionContext.class),
             Providers.of(versionRepo),
             mockSettingsManifest,
             programCache,
+            programDefCache,
             versionsByProgramCache);
   }
```

Comment:
question (blocking): Is it worth adding a short test here that verifies the new set & get methods work? (And maybe that they don't do anything if the flag is off?)

Reply:
Yep good call - added!